{"导致测试失败的例外信息":[19988481,41353217,46530561,57868289],"导致测试跳过的例外信息":[21233665,42008577,50397185,59047937],"导出类型到自身类型和继承的类型":[38207489,66453505]}